nemo/collections/common/parts/preprocessing/collections.py

```diff
@@ -198,8 +208,9 @@ def __init__(self, manifests_files: Union[str, List[str]], *args, **kwargs):
             offsets.append(item['offset'])
             speakers.append(item['speaker'])
             orig_srs.append(item['orig_sr'])
+            langs.append(item["lang"])
```

Comment:
'lang' - and also what if there's no lang field in the tokenizer/ manifest?

Reply:
i think is resolved per below, we use nones. double quotes changed to single quotes